content/tutorials/webrtc/basics/en/index.html

```diff
@@ -27,125 +27,127 @@
 
-<p>WebRTC implemented open standards for real-time, plugin-free video, audio and data communication. The need was real:</p>
+<p>WebRTC implemented open standards for real-time, plugin-free video, audio, and data communication. The need was real:</p>
 
   <ul>
-    <li>Many web services used RTC, but needed downloads, native apps or plugins. Those included Skype, Facebook and Google Hangouts.</li>
-    <li>Downloading, installing and updating plugins is complex, error prone and annoying.</li>
-    <li>Plugins are difficult to deploy, debug, troubleshoot, test and maintain&mdash;and may require licensing and integration with complex, expensive technology. It's often difficult to persuade people to install plugins in the first place!</li>
+    <li>Many web services used RTC, but needed downloads, native apps, or plugins. These included Skype, Facebook, and Hangouts.</li>
+    <li>Downloading, installing, and updating plugins is complex, error prone, and annoying.</li>
+    <li>Plugins are difficult to deploy, debug, troubleshoot, test, and maintain&mdash;and may require licensing and integration with complex, expensive technology. It's often difficult to persuade people to install plugins in the first place!</li>
   </ul>
 
-<p>The guiding principles of the WebRTC project are that its APIs should be open source, free, standardized, built into web browsers and more efficient than existing technologies.</p>
+<p>The guiding principles of the WebRTC project are that its APIs should be open source, free, standardized, built into web browsers, and more efficient than existing technologies.</p>
 
 <h2 id="toc-where">Where are we now?</h2>
-<p>WebRTC is used in various apps like WhatsApp, Facebook Messenger, appear.in and platforms such as TokBox. WebRTC has also been integrated with <a href="https://labs.ericsson.com/developer-community/blog/beyond-html5-conversational-voice-and-video-implemented-webkit-gtk" title="Ericsson article about WebKitGTK+">WebKitGTK+</a> and <a href="https://www.youtube.com/watch?v=Vm5ebKWKNE8" title="basysKom showcasing WebRTC based video chat in QML application">Qt</a> native apps.</p>
+<p>WebRTC is used in various apps, such as WhatsApp, Facebook Messenger, and appear.in, as well as platforms, such as TokBox. WebRTC has also been integrated with <a href="https://labs.ericsson.com/developer-community/blog/beyond-html5-conversational-voice-and-video-implemented-webkit-gtk" title="Ericsson article about WebKitGTK+">WebKitGTK+</a> and <a href="https://www.youtube.com/watch?v=Vm5ebKWKNE8" title="basysKom showcasing WebRTC based video chat in QML application">Qt</a> native apps.</p>
```

Comment:
We might want to update this depending on what WebRTC team would like us to point to.

Reply:
I'll adjust.